x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/entsearch/analytics/behavioral_analytics-events-final_pipeline.json

```diff
@@ -57,13 +57,6 @@
         "ignore_missing": true
       }
     },
-    {
```

Comment:
@eyalkoren May I ask a few questions? This pipeline is now managed by the [IndexTemplate registry ](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/analytics/AnalyticsTemplateRegistry.java#L85-L94) 

- do I need to increase the `REGISTRY_VERSION` for this change?

Reply:
Generally speaking- yes. As long as you use the same pipeline ID, I believe it won't be upgraded if it is already installed otherwise. 
In your specific case, maybe it is not required (and even not preferred), because the geoip DB has already been downloaded, so you might as well keep using its processor...

I guess you need to decide whether you enforce proper version maintenance (and increase the version) or make an exception in this case (not increase the version- then only new clusters will use this form).
I hope this makes sense...

Reply:
Thanks for the answer! Yep as this is only a temporary change and will have a further change before 8.9 release, I will keep the version number the same.

Reply:
So if your further change restores the pipeline state to be the same as it was before, this is indeed not required.
However, if after all your pipeline (or any other component installed through your registry) changes, then version should be increased, otherwise it won't be upgraded in rolling upgrades.